packages/devextreme/js/__internal/grids/pivot_grid/field_chooser/m_field_chooser.ts

```diff
@@ -587,17 +590,17 @@ const FieldChooser = FieldChooserBase.inherit({
       $fieldsContainer.addClass(CLASSES.treeView.borderVisible);
       that._renderFieldsTreeView($fieldsContainer);
     }
-  },
+  }
 
   _getSortableOptions() {
     // TODO
     return {
-
+      direction: '',
```

Comment:
It's may be dangerous change
But if you sure that it's okay let's leave it as is :)